server/src/domain/user/user.service.ts

```diff
@@ -61,6 +61,7 @@ export class UserService {
 
   async update(auth: AuthDto, dto: UpdateUserDto): Promise<UserResponseDto> {
     await this.findOrFail(dto.id, {});
+    await this.userRepository.syncUsage();
```

Comment:
Quotas might enable servers with lots of users. It might be worth making it possible to only sync a specific user.

Also, would it make sense to only run this if the quota is updated, not anytime any user related thing changes?